app/pages/notifications/notification-section.jsx

```diff
@@ -32,30 +32,29 @@ export default class NotificationSection extends Component {
       });
     } else {
       apiClient.type('projects').get(this.props.projectID, { include: 'avatar' })
-      .catch((error) => {
-        this.setState({ error });
-      })
-      .then((project) => {
-        if (project.links.avatar) {
-          apiClient.type('avatars').get(project.links.avatar.id)
-          .then((avatar) => {
-            this.setState({
-              name: project.display_name,
-              avatar: avatar.src
-            })
-            .catch(() => {
-              this.setState({
-                name: project.display_name
+        .catch((error) => {
+          this.setState({ error });
+        })
+        .then((project) => {
+          if (project.links.avatar) {
+            apiClient.type('avatars').get(project.links.avatar.id)
+              .catch(() => {
+                this.setState({
+                  name: project.display_name
+                });
+              })
```

Comment:
Won't this catch block crash the following then block, because it doesn't pass an avatar into it?

Reply:
I'm not really sure what this block is doing because it's setting state, but the then block sets state again.

If it's supposed to catch errors in the API request, it should return a default avatar for the setState call in the then block.

Reply:
I moved the catch below the then block, similar to before, though before I was getting linter/console warnings.